### PR TITLE
EP11: fixes for CKM_IBM_BTC_DERIVE 

### PR DESCRIPTION
**Pull request by ifranzki**
- pass back chain code in mech param
- supply SPKI when deriving public key
- supply SPKI for private key import (all asym key types)

### CODE REVIEW

File: usr/lib/ep11_stdll/ep11_specific.c [hunk 1]
```diff
@@ -3746,9 +3755,13 @@ CK_RV token_specific_object_add(STDLL_TokData_t * tokdata, SESSION * sess,
     CK_ATTRIBUTE *attr = NULL;
     CK_BYTE blob[MAX_BLOBSIZE];
     size_t blobsize = sizeof(blob);
+    CK_BYTE spki[MAX_BLOBSIZE];
+    CK_ULONG spkisize = sizeof(spki);
     CK_RV rc;
```
Comment (jschmidb) [inline usr/lib/ep11_stdll/ep11_specific.c:3760]: From review together with Vishnu: here you are using CK_ULONG spkisize, but in the functions import_xxxxx_key() you are using size_t *spki_size. Yes, it's the same, but just to mention. For comparison: blob_size is a size_t at both places.

Comment (ifranzki) [reply usr/lib/ep11_stdll/ep11_specific.c]: Ah right. Will change it to size_t here as well (for consistency)

**Review by jschmidb — APPROVED** [review]
lgtm

**Review by vkanjur — APPROVED** [review]
Approved

**Pull request merged**